### PR TITLE
[small]feature/318

### PR DESCRIPTION
**Pull request by kaminuma**
- I can see a Phone field in the user create form
add form
- Format should be + country code and 10 phone number digits
Implemented so that a warning message appears when blurring numbers other than 10 digits.
- Phone field should be located after Lastname field and before Email field
Implemented same as above
- +52 should be pre-populated since it is the country with most of our users.
Create another form and set it to 52 by default. If the user is from another country, this can be changed manually.

Cautionary note
Since the phone number and country code are separated, it is necessary to share their recognition on the backend side.

### CODE REVIEW

File: src/main/java/com/josdem/vetlog/command/UserCommand.java [hunk 1]
```diff
@@ -38,6 +38,12 @@ public class UserCommand implements Command {
     @Size(min = 1, max = 100)
     private String lastname;
 
+    @Size(min = 1, max = 250)
+    private String phone;
+
+    @Size(min = 1, max = 250)
+    private String countryCode;
+
```
Comment (josdem) [inline src/main/java/com/josdem/vetlog/command/UserCommand.java:46]: Based on this information: https://countrycode.org/ max string lenght for country codes is 5, therefore 250 seems too large. However, no worries we will take care of this lenght in this issue: https://github.com/josdem/vetlog-spring-boot/issues/319

File: src/main/resources/i18n/messages.properties [hunk 2]
```diff
@@ -65,6 +65,7 @@ user.view.create.label=Please, provide your information
 user.view.create.title=Create User
 user.firstname=Name
 user.lastname=Last Name
+user.phone=Phone Number
 user.email=Email
```
Comment (josdem) [inline src/main/resources/i18n/messages.properties:69]: Currently we support EN and ES languages, therefore we need to add `user.phone` in this file: 
https://github.com/josdem/vetlog-spring-boot/blob/main/src/main/resources/i18n/messages_es.properties
Translation should be: Número Telefónico

**Review by josdem — CHANGES_REQUESTED** [review]
@kaminuma  Only adding `user.phone` to the https://github.com/josdem/vetlog-spring-boot/blob/main/src/main/resources/i18n/messages_es.properties file is required before we can merge.  Good job!

File: src/main/java/com/josdem/vetlog/command/UserCommand.java (hunk 1 above)
Comment (kaminuma) [reply src/main/java/com/josdem/vetlog/command/UserCommand.java:46]: That's certainly true. thank you. 
Please take action using 319

File: src/main/resources/i18n/messages.properties (hunk 2 above)
Comment (kaminuma) [reply src/main/resources/i18n/messages.properties:69]: @josdem 
I tried adding it to the Spanish file, but please check if there is a problem mm

Comment (josdem) [reply src/main/resources/i18n/messages.properties:69]: Looks good now 👍

**Review by josdem — APPROVED** [review]
Thanks @kaminuma  for doing suggested changes!

**Pull request merged**